shared/src/main/java/com/couchbase/lite/LogDomain.java

```diff
@@ -21,5 +21,5 @@
  * Log domain. The log domains here are tentative and subject to change.
  */
 public enum LogDomain {
-    ALL, DATABASE, QUERY, REPLICATOR, NETWORK
+    ALL, COUCHBASE, DATABASE, QUERY, REPLICATOR, NETWORK
```

Comment:
You told me once but I forgot. When will the COUCHBASE domain be used?

After thought, can we use DATABASE domain for that so that we don't need to add a new domain here?

Reply:
I couldn't find where it get used in this PR as well.

Reply:
COUCHBASE will be the default domain for unmapped LiteCore domains.  It is used in the JNI (coincidentally one of the problems Sandy ran into was "COUCHBASE not defined")

Reply:
Can we do DATABASE instead so we don't need to add a new domain?

Reply:
This might now answer the question. The weird part is that as a user I don't know when I should include COUCHBASE in the domain list. It seems like it is a domain that should be included all the time because it's something that might get logged randomly?

Reply:
Is it something that can be handled case by case to manually catch and map to one of the existing domains?

Reply:
I guess it's more of a side effect of giving the user the choice at all.  How many will simply just change the log level and not bother with domains?  Ultimately I think they should all be removed (maybe even internally) to make things simpler so I guess I can put up with Database being "odd" for now and we can answer the question properly in 3.0.

Reply:
Totally agree.

Reply:
I still want to come up with a mapping so that not everything gets dumped into database.  I'll write one today.